VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesFragment.java

```diff
@@ -148,4 +122,110 @@ public void onClickAddComment(@NonNull DiscussionComment comment) {
     public void onClickViewComments(@NonNull DiscussionComment comment) {
         router.showCourseDiscussionComments(getActivity(), comment, discussionThread.isClosed());
     }
+
+    public static class DiscussionResponsesLoader implements
+            InfiniteScrollUtils.PageLoader<DiscussionComment> {
+        private InfiniteScrollUtils.PageLoadCallback<DiscussionComment> callback;
+
+        private Context context;
```

Comment:
Also my suggestion would be to group the three parameter storing variables together (i.e. `context`, `threadId` and `isQuestionTypeThread`).